http_server/queries/servers/servers_select_highest_port.php

```diff
@@ -2,20 +2,19 @@
 
 function servers_select_highest_port($pdo)
 {
-    $stmt = $pdo->prepare('
-        SELECT servers.port
-        FROM servers
-        ORDER BY servers.port DESC
-        LIMIT 1
+    $result = $pdo->exec('
+        SELECT port
+          FROM servers
+         LIMIT 1
```

Comment:
without an `ORDER BY port DESC` line, this may not return the highest port #

Reply:
u rite, I'll fix that